crypto/ledger/ledger_secp256k1.go

```diff
@@ -210,11 +212,17 @@ func warnIfErrors(f func() error) {
 }
 
 func convertDERtoBER(signatureDER []byte) ([]byte, error) {
-	sigDER, err := btcec.ParseDERSignature(signatureDER, btcec.S256())
+	sigDER, err := ecdsa.ParseDERSignature(signatureDER)
 	if err != nil {
 		return nil, err
 	}
-	sigBER := tmbtcec.Signature{R: sigDER.R, S: sigDER.S}
+
+	sigStr := sigDER.Serialize()
+	var r, s *big.Int
+	r.SetBytes(sigStr[:32])
+	s.SetBytes(sigStr[32:64])
```

Comment:
This doesn't seem right. The format of `Signature.Serialize` is

https://github.com/decred/dcrd/blob/0e7920ce30257aa1aced00e34ac637423f50965e/dcrec/secp256k1/ecdsa/signature.go#L69-L86

In particular, the R and S values are encoded (AFAICT) at index 4 and 7 respectively.

Reply:
Right, updated!